go-update/src/main/java/org/reactome/release/goupdate/GoTermInstanceModifier.java

```diff
@@ -238,17 +254,17 @@ public void updateReferrersDisplayNames() throws Exception
 		
 		if (instance.getSchemClass().getName().equals(ReactomeJavaConstants.GO_BiologicalProcess))
```

Comment:
This *might* be made shorter if it was `instance.getSchemClass().isa(ReactomeJavaConstants.GO_BiologicalProcess)`, assuming that GO_BiologicalProcess/CellularComponent/MolecularFunction are always GKInstances

Reply:
Yes, I think "isa" is a beter way to handle this, in general.